dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -505,12 +505,13 @@ public Set<HealthCheckResult> checkHealth(
                     .map(result -> new HealthCheckResult(result.getKey(), result.getValue().isHealthy()))
                     .collect(Collectors.toSet());
         } else {
+            // Alarms may depend on the syntax of this logging statement, so make sure to check if you change it.
             results.entrySet().stream()
                     .filter(result -> !result.getValue().isHealthy())
                     .forEach(result -> LOG.error("Health check '{}' failed with error: {}", result.getKey(), result.getValue().getMessage()));
             String failedHealthCheckNames = results.entrySet().stream()
                     .filter(result -> !result.getValue().isHealthy())
-                    .map(result -> String.format("'%s'", result))
+                    .map(result -> String.format("'%s'", result.getKey()))
```

Comment:
The original code looks like it was trying to create a list of names, but it was calling `result.toString`, which also included the result message and some other information.  In some cases, we don't want the details of a health check failure to be public, thus the above change.